src/components/DoubleLine.js

```diff
@@ -20,4 +21,11 @@ const GreenLine = () => (
   </_DoubleLineContainer>
 )
 
-export { GreenLine }
+const BlueLine = () => (
+  <_DoubleLineContainer>
+    <div className="w-75">
+      <_DoubleLine src={double_line_blue} />
+    </div>
+  </_DoubleLineContainer>
+)
+export { GreenLine, BlueLine }
```

Comment:
Could we maybe change this to a generic ColouredLine function that takes the specific src as a variable?

Reply:
@lucymk fair point! I've addressed this in the next PR